qcut/docs/issues/video-ai/add-wan25-image-to-video.md

```
@@ -0,0 +1,1595 @@
+- [ ] **UI Control: Prompt expansion toggle** enabled by default
+- [ ] **Dynamic cost display** updates based on resolution + duration selection:
+  - [ ] 480p: $0.05/s
+  - [ ] 720p: $0.10/s
+  - [ ] 1080p: $0.15/s
+- [ ] Audio synchronizes with generated video when provided
+- [ ] Generated video matches user-selected resolution and duration
+
+## Files to Modify Summary
+
+### 1. `qcut/apps/web/src/components/editor/media-panel/views/ai-constants.ts`
+   - **Location**: `AI_MODELS` array, after `ltxv2_fast_i2v`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)